gdpr/utils.py

```diff
@@ -51,18 +51,30 @@ def get_field_or_none(model: Type[Model], field_name: str):
 
 def get_reversion_versions(obj: Any) -> QuerySet:
     from reversion.models import Version
-    from django.contrib.contenttypes.models import ContentType
 
     return Version.objects.get_for_object(obj)
 
 
+def get_auditlog_entries(obj: Any) -> QuerySet:
+    from auditlog.models import LogEntry
+    from django.contrib.contenttypes.models import ContentType
+
+    content_type_id = ContentType.objects.get_for_model(obj).id
+    return LogEntry.objects.filter(object_pk=obj.pk, content_type=content_type_id)
+
+
 def get_reversion_version_model(version) -> Type[Model]:
     """Get object model of the version."""
     if hasattr(version, '_model'):
         return version._model
     return version.content_type.model_class()
 
 
+def get_auditlog_entry_model(entry) -> Type[Model] | None:
```

Comment:
`entry` is missing type annotation.

Reply:
I cant, in both cases, since it's `LogEntry`, which is optional dependency (there might be reversion or auditlog), so I cannot import it.

Reply:
Because it wont be true, that function wont work with general `Model`, not each Model has `content_type` field

Reply:
How about using something like this?

```
class HasContentType(Protocol):
    content_type: ContentType


def get_auditlog_entry_model(entry: HasContentType) -> Type[Model] | None:
    ...
```

I believe this satisfies `mypy` but I'm not sure this is best way.

Reply:
I chose different solution, have a look :-)

Reply:
Looks great! Thanks.